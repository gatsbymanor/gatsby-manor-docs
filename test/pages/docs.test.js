import React from 'react';
import Docs from '../../src/pages/docs';

describe('<Docs />', () => {
  var testData = {
    allMarkdownRemark: {
      edges: []
    }
  }

  const component = shallow(
    <Docs data={testData}/>
  );

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
