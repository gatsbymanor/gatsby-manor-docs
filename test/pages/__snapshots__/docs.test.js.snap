// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Docs /> should render correctly 1`] = `
<div>
  <div
    className="docs--banner"
  >
    <h1
      className="docs--banner--title"
    >
      Docs
    </h1>
  </div>
  <div
    className="container mr-auto docs--section"
  >
    <div>
      <h1
        className=""
      >
        Quick Start
      </h1>
    </div>
    <div>
      <h2
        className=""
      >
        Github
      </h2>
      <p>
        Any questions about the Jay, Themes, or Gatsby Manor can be directed to 
        <a
          href="https://github.com/gatsbymanor"
        >
          Github
        </a>
        . Find the repo you are interested in and create an issue detailing your problem. A core member will join the conversation when they are free. PR's are welcome for easy fixes or brand new features!
      </p>
    </div>
  </div>
</div>
`;
